index.html

```diff
@@ -255,43 +255,304 @@ <h2 class="section-title">Components</h2>
                 
 
                 
-                    <div class="icons-section" id="icons-example">
-                        <h2>Icons</h2>
+                    <div class="section icons-section" id="icons-example">
+                        <h2 class="section-title">Icons</h2>
                         
+
+    <h3 class="section-sub-title">Icon usage</h3>
+
+    
```

Comment:
@hippogriffic You can add it to `.gitignore` so it's ignored, then merge into `gh-pages` and remove from `.gitignore`, then commit it there. That means you can compile in `master` and not have to commit it, but then when merging into `gh-pages` you compile it and commit.